MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,72 +1,312 @@
+// swiftlint:disable all
+
 import UIKit
 
 final class MovieQuizViewController: UIViewController {
-    // MARK: - Lifecycle
+    
+    @IBOutlet private var imageView: UIImageView!
+    @IBOutlet private var textLabel: UILabel!
+    @IBOutlet private var counterLabel: UILabel!
+    @IBOutlet private var noButton: UIButton!
+    @IBOutlet private var yesButton: UIButton!
+    
+    @IBAction private func noButtonClicked(_ sender: UIButton) {
+        processUserAnswer(answer: false)
+    }
+    @IBAction private func yesButtonClicked(_ sender: UIButton) {
+        processUserAnswer(answer: true)
+    }
+    
+    private var currentQuestionIndex: Int = 0
+    private var currentQuestion: QuizQuestion { questions[currentQuestionIndex] }
+    private var analytic: QuizAnalytics = QuizAnalytics()
+    
+    
+    // MARK: - LIFECYCLE
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        show(quiz: convert(model: currentQuestion))
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        imageBorderDefaultStyle() // Настройка стиля обводки ImageView
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+    }
+    
+    
+    // MARK: - QUIZ STEP
+    
+    private struct QuizQuestion {
+        let image: String
+        let text: String
+        let correctAnswer: Bool
+    }
+    
+    private struct QuizStepViewModel {
+        let image: UIImage
+        let question: String
+        let questionNumber: String
+    }
+    
+    // Конвертор данных вопроса в данные для заполнения вью
+    private func convert(model: QuizQuestion) -> QuizStepViewModel {
+        let quiz: QuizStepViewModel = QuizStepViewModel(
+            image: UIImage(named: model.image)!, // TODO: подставить дефолтную картинку
```

Comment:
Давай чтобы не форс анврапить, просто вот такую заглушку хотя бы сделаем
```suggestion
            image: UIImage(named: model.image) ?? UIImage(),
```